### PR TITLE
Fix lint problems + Fix tests

### PR DESCRIPTION
**Pull request by Klynger**
#### What does this PR do? \*

Add lint configuration + Fix lint problems + Fix tests + Update dependencies

#### How to test it? \*

https://checkbox--unileverb2b.myvtex.com/#
image
Click in `Registrar`
It should run

The tests are not working yet, so I need some help guys

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -6,6 +6,14 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
+### Added
+- Eslint configuration.
+
+### Fixed
+- Test errors.
+
+### Securytiy
```
Comment (iaronaraujo) [inline CHANGELOG.md:15]:
```suggestion
### Security
```

**Comment by kaisermann** [comment]
Waiting for the next release of `@typescript-eslint/eslint-plugin` via https://github.com/typescript-eslint/typescript-eslint/issues/2009

File: react/components/Checkbox.tsx [hunk 2]
```diff
@@ -30,7 +30,7 @@ export const Checkbox: FC<{
           {...(checked ? { checked: true } : { checked: false })}
           onChange={() => {
             const pointer = checkboxObject.pointer
-
+            console.log('styleguide checkbox - chamou boy')
```
Comment (iaronaraujo) [inline react/components/Checkbox.tsx:33]:
```suggestion
```
🌚

File: react/components/Checkbox.tsx [hunk 3]
```diff
@@ -55,7 +55,7 @@ export const Checkbox: FC<{
             {...(checked ? { checked: true } : { checked: false })}
             onChange={() => {
               const pointer = checkboxObject.pointer
-
+              console.log('styleguide checkbox - chamou boy')
```
Comment (iaronaraujo) [inline react/components/Checkbox.tsx:58]:
```suggestion
```

**Pull request merged**